content/docs/alerting/configuration.md

```diff
@@ -64,12 +64,17 @@ global:
   # The default SMTP From header field.
   [ smtp_from: <tmpl_string> ]
   # The default SMTP smarthost used for sending emails.
+  # Provide the correct port for the smarthost (25 / 587 for SMTP over TLS (STARTTLS), 465 for deprecated SMTPS). 
```

Comment:
Is this field for the smarthost or the port? I can see this confusing users.

"The default SMTP smarthost used for sending emails, including port number." would be clearer.